systemdev/kernel-pinephone.rst

```diff
@@ -0,0 +1,71 @@
+PinePhone Kernel
+================
+
+This document describes how to build and install the PinePhone kernel. Before getting to the actual compilation, we describe the PinePhone recovery. At the bottom there are some references relevant for kernel development.
```

Comment:
Since this page is linked to via both the PinePhone and PineTab text, it might be relevant to name it accordingly.

Reply:
right. I'm walking the fine line between hearing that it's the same, but not having a pinetab to actually check any of this. 

I've now changed the title and intro statement. However in the rest of the doc I say "PinePhone" ... I hope this is safe and clear enough for everyone?! Alternatively, if you tell me "nah, it all applies", I could be more generic and say "device", or, the opposite I could be more careful and put some preamble that all of this more or less should apply to tab aswell, but has only been tested wtih phone ...